R/deploy.R

```diff
@@ -487,49 +487,65 @@ has_image <- function(content) {
 #'
 #' Set the Content Image using a variety of methods.
 #'
-#' NOTE: `set_image_webshot()` requires [webshot2::webshot()], but currently
-#' skips and warns for any content that requires authentication until the
-#' `webshot2` package supports authentication.
-#'
 #' @param content A content object
-#' @param path The path to an image on disk
-#' @param url The url for an image
-#' @param ... Additional arguments passed on to [webshot2::webshot()]
+#' @param path A file path or URL to an image
 #'
-#' @rdname set_image
 #' @family content functions
+#' @rdname set_content_image
 #' @export
-set_image_path <- function(content, path) {
-  warn_experimental("set_image_path")
+set_content_image <- function(content, path) {
```

Comment:
Could we treat `path` similarly to `read.csv()`, `readr::read_csv()`, and other functions, where the `file` argument could be a connection as well as a file name or URL?

Consider when the caller already has image data in memory.

Reply:
This seems sensible.